src/options_parser.cc

```diff
@@ -35,11 +39,30 @@ OptionsResult* protagonist::ParseOptionsObject(Handle<Object> optionsObject) {
             else
                 optionsResult->options &= snowcrash::ExportSourcemapOption;
         }
+        else if (TypeOptionKey == *String::Utf8Value(key)) {
+            // TypeOption
+            char *strValue = *String::Utf8Value(value);
+            if (TypeOptionAst == strValue) {
+                // TODO
+                // optionsResult->options &= snowcrash::TypeOptionAst;
```

Comment:
I'm a little confused about this. Protagonist is already using Drafter, via the `Drafter::ParseBlueprint` function that takes a [SnowCrash options object](https://github.com/apiaryio/drafter/blob/pksunkara/refract-ast/src/drafter.cc#L16-L18). Where exactly will the setting be passed to Drafter? I must be missing something.